extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -20,6 +20,16 @@ public function __construct() {
 	}
 
 	/**
+	 * Check if a user is an admin
+	 *
+	 * @param string $group The user's group from ug_group
+	 * @return bool
+	 */
+	public function isAdmin( $group ) {
+		return strpos( $group, 'sysop' ) !== false;
```

Comment:
This was the original check. The function is called once for each row in a query result, with $row->ug_group. This fix was made on the assumption that ug_groups contains all the user's group space separated, which I think is not correct.

We get the data by joining wikia_user_properties and user_groups, so I guess that means that for users who have more than one group, we just get a random one.

Is there an easy and low-cost way to get all of a user's groups or an isAdmin flag when we have the user id?